psrp/script.yaml

```
@@ -5042,7 +5042,7 @@
   fr: |
     Voussa pouvoir être téléportés au 10ème sous-sol de la grotte Dézoris!<wait more>
     Pas blague. Noussa pas mentir à voussa.<wait>
-  pt-br: Há um atalho no décimo nível subterrâneo na Caverna de Dezóris. Não estou brincando.<wait>
+  pt-br: Há uma passagem secreta no décimo nível subterrâneo na Caverna de Dezóris. Não estou brincando.<wait>
```

Other languages finish the "I'm not joking!" with an exclamation mark which fits the tone of the phrase. Suggest using it in pt-BR too.
```suggestion
  pt-br: Há uma passagem secreta no décimo nível subterrâneo na Caverna de Dezóris. Não estou brincando!<wait>
```